documentation/source/overview/studies.rst

```diff
@@ -5,104 +5,134 @@ Studies using SCT
+    -  Meyer *et al.* `Optimized cervical spinal cord perfusion MRI after traumatic injury in the rat. <https://journals.sagepub.com/doi/10.1177/0271678X20982396>`_ **J. of Cerebral Blood Flow & Metabolism** 2021
+    -  Solanes *et al.* `3D patient-specific spinal cord computational model for SCS management: potential clinical applications. <https://pubmed.ncbi.nlm.nih.gov/33556926/>`_ **Journal of Neural Engineering** 2021
+    -  Johnson *et al.* `Changes in White Matter of the Cervical Spinal Cord after a Single Season of Collegiate Football. <https://www.liebertpub.com/doi/10.1089/neur.2020.0035>`_ **Neurotrauma Reports** 2021
+    -  Ost *et al.* `Spinal Cord Morphology in Degenerative Cervical Myelopathy Patients; Assessing Key Morphological Characteristics Using Machine Vision Tools. <https://www.mdpi.com/2077-0383/10/4/892>`_ **Journal of Clinical Medicine** 2021
+    -  Lee *et al.* `Diffusion‐prepared fast spin echo for artifact‐free spinal cord imaging. <https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.28751>`_ **Magnetic Resonance in Medicine** 2021
+    -  Tinnermann *et al.* `Observation of others’ painful heat stimulation involves responses in the spinal cord. <https://pubmed.ncbi.nlm.nih.gov/33789899/>`_ **Science Advances** 2021
+    -  Zhang *et al.* `Automatic spinal cord segmentation from axial-view MRI slices using CNN with grayscale regularized active contour propagation. <https://pubmed.ncbi.nlm.nih.gov/33780869/>`_ **Computers in Biology and Medicine** 2021
+    -  Savini *et al.* `Pilot Study on Quantitative Cervical Cord and Muscular MRI in Spinal Muscular Atrophy: Promising Biomarkers of Disease Evolution and Treatment? <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8039452/>`_ **Front Neurol** 2021
+    -  Vallotton *et al.* `Tracking white and grey matter degeneration along the spinal cord axis in degenerative cervical myelopathy. <https://pubmed.ncbi.nlm.nih.gov/34238034/>`_ **J Neurotrauma** 2021
+    -  Shahrampour *et al.* `Atlas-Based Quantification of DTI Measures in a Typically Developing Pediatric Spinal Cord. <https://pubmed.ncbi.nlm.nih.gov/34326104/>`_ **American Journal of Neuroradiology.** 2021
+    -  Adibi *et al.* `Quantitative Magnetic Resonance Imaging Analysis of Early Markers of Upper Cervical Cord Atrophy in Multiple Sclerosis and Neuromyelitis Optica Spectrum Disorder. <https://pubmed.ncbi.nlm.nih.gov/34306756/>`_ **Mult Scler Int.** 2021
+    -  Martucci *et al.* `Spinal Cord Resting State Activity in Individuals With Fibromyalgia Who Take Opioids. <https://www.frontiersin.org/articles/10.3389/fneur.2021.694271/full>`_ **Front. Neurol.** 2021
+    -  David *et al.* `Longitudinal changes of spinal cord grey and white matter following spinal cord injury. <https://jnnp.bmj.com/content/early/2021/08/11/jnnp-2021-326337.info>`_ **Journal of Neurology, Neurosurgery & Psychiatry.** 2021
+
```

Comment:
@copperwiring I believe the conflicts were due to https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3517 being merged first, which added 4 new references.

But, when resolving the conflicts in d3a81cd, I think that those 4 new references got discarded.

So, could you re-add the references (with your new formatting)? :slightly_smiling_face:

Reply:
Aaah, I see. I was wondering what was going on. Thanks for checking. On it.

Reply:
Fixed!